platforms/gradle/foundry-gradle-plugin/src/main/kotlin/foundry/gradle/StandardProjectConfigurations.kt

```diff
@@ -510,6 +510,7 @@ internal class StandardProjectConfigurations(
                 .getOrElse(false)
           val isAndroidTestEnabled = variant is HasAndroidTest && variant.androidTest != null
           if (isAndroidTestEnabled) {
+            dependencies.add("androidTestUtil", "androidx.test:orchestrator")
```

Comment:
Before you merge, we have a property for this that we should probably still gate this on just to be safe. `FoundryProperties.useOrchestrator`

Reply:
I think we keep it on by default and have the killswitch in place since we already have the property available anyway, doesn't hurt to have around